src/scripts/main.js

```diff
@@ -1 +1,105 @@
 'use strict';
+
+const firstPromise = new Promise((resolve, reject) => {
+  let isResolved = false;
+
+  const clickHandler = () => {
+    if (!isResolved) {
+      isResolved = true;
+      clearTimeout(timer);
+      document.removeEventListener('click', clickHandler);
+
+      resolve('First promise was resolved');
+    }
+  };
+
+  document.addEventListener('click', clickHandler);
+
+  const timer = setTimeout(() => {
+    if (!isResolved) {
+      isResolved = true;
+      document.removeEventListener('click', clickHandler);
+
+      reject(new Error('First promise was rejected'));
+    }
+  }, 3000);
+});
+
+firstPromise
+  .then((message) => {
+    const newElement = document.createElement('div');
+
+    newElement.setAttribute('data-qa', 'notification');
+    newElement.classList.add('success');
+    newElement.innerText = message;
+    document.body.appendChild(newElement);
+  })
+  .catch((error) => {
+    const newElement = document.createElement('div');
+
+    newElement.setAttribute('data-qa', 'notification');
+    newElement.classList.add('error');
+    newElement.innerText = error;
+    document.body.appendChild(newElement);
+  });
+
+const secondPromise = new Promise((resolve, reject) => {
+  let isResolved = false;
```

Comment:
As with 'firstPromise', the 'isResolved' variable is unnecessary for 'secondPromise' as well. You can remove it and rely on the promise's state.